executable-spec/src/Cardano/Ledger/Generators/QuickCheck.hs

```diff
@@ -80,17 +82,31 @@ boundedWith aMinBound aMaxBound maxDifference =
   where mid = aMinBound + (aMaxBound - aMinBound) `div` 2
 
 prvNoQuorum :: Gen Word8
-prvNoQuorum = Gen.choose (3, 7)
+prvNoQuorum =  Gen.choose (1, 3)
```

Comment:
```suggestion
prvNoQuorum = Gen.choose (1, 3)
```

Reply:
Don't we want to test what happens if this generator returns `0`?

Reply:
Done OK